pyproject.toml

```diff
@@ -21,3 +21,7 @@ include = [".entangled/tasks.json"]
 [tool.brei.runner.julia]
 command = "julia"
 args = ["--project=workenv", "${script}"]
+
+[[tool.brei.task]]
+runner = "bash"
+script = "mkdir docs/src/_fig; cp -r docs/src/fig/* docs/src/_fig"
```

Comment:
You'd want to create the `fig` directory, not the `_fig` one. And use `-p` to not throw an error if it already exists. For copying it is best to behave a bit `rsync` like. I'd recommend using `cp -pur` (*p*reserve dates and flags, *u*pdate only newer files and do so *r*ecursively).

Reply:
also, should we remove any figures that are no longer existing?

Reply:
Maybe `rsync -avh docs/src/_fig/ docs/src/fig/ --delete` is the better command.

Reply:
The task should have a name, so that you can run `brei copy_figures`. Add `name="copy_figures"`.

Reply:
Also, the task should depend on the `figures` task to make sure figures are up-to-date.

add the line `requires=["#figures"]`.

Reply:
> You'd want to create the `fig` directory, not the `_fig` one. And use `-p` to not throw an error if it already exists. For copying it is best to behave a bit `rsync` like. I'd recommend using `cp -pur` (_p_reserve dates and flags, _u_pdate only newer files and do so _r_ecursively).

Rather, both folders should already exist, since `fig` is git tracked and `_fig` is where figures are written to. Make the directory `_fig` and touch a file `.gitignore` inside it. That way the folder `_fig` exists for everyone but its contents aren't tracked.